source/server/filter_chain_manager_impl.cc

```diff
@@ -1,6 +1,10 @@
 #include "source/server/filter_chain_manager_impl.h"
 
 #include "envoy/config/listener/v3/listener_components.pb.h"
+#include "envoy/event/dispatcher.h"
```

Comment:
Given the risk of the HCM change, I would really rather you split this part of the change into a separate PR, but I'm not going to demand it if you don't feel that it's worth it.

Reply:
Responded in top-level comment [here](https://github.com/envoyproxy/envoy/pull/16201#pullrequestreview-750767228)